### PR TITLE
Support --google-disk-type flag for google driver

### PR DESCRIPTION
**Pull request by zchee**
Signed-off-by: Koichi Shiraishi <zcheeee@gmail.com>

### CODE REVIEW

**Comment by zchee** [comment]
cc @ehazlett

**Comment by zchee** [comment]
sorry, I forgot gofmt.

**Comment by zchee** [comment]
Does this commit there any problem?
Because not familiar with golang, please tell me if there is a problem.

Thanks.

**Comment by ehazlett** [comment]
@zchee Thanks for the contribution!   Looks like this needs a rebase and signed commits: https://github.com/docker/docker/blob/master/CONTRIBUTING.md#sign-your-work

**Comment by zchee** [comment]
@ehazlett Fiinish Signed-off.

**Comment by ehazlett** [comment]
@zchee thx -- this needs a rebase

**Comment by zchee** [comment]
Rebased.

**Comment by ehazlett** [comment]
@zchee this also needs a `gofmt` https://travis-ci.org/docker/machine/builds/54972776#L208  thanks

**Comment by zchee** [comment]
I apologize for the delay in replying.
Fixed.

File: drivers/google/google.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ type Driver struct {
 	SSHPort        int
 	Zone           string
 	MachineType    string
+	DiskTypeURL    string
```
Comment (ehazlett) [inline drivers/google/google.go]: Sorry to be nitpicking, but why is this called `diskTypeURL`?  Shouldn't we be just specifying the type?

**Comment by zchee** [comment]
@ehazlett 
I wrote golang this is the first time, not well understood golang... :(
I will study harder.

What to you is what improvements have think ideal?

**Comment by ehazlett** [comment]
No problem!  I was just wondering why the variables aren't matching the option names.  Seems like it would be confusing down the road.

**Comment by zchee** [comment]
Fixed.
Will that be OK?

**Comment by ehazlett** [comment]
LGTM

**Comment by zchee** [comment]
Thanks :tada:

**Comment by ehazlett** [comment]
thanks for the contribution!!

**Pull request merged**